src/styling/ApartmentProtectedIndex.css

```diff
@@ -1,6 +0,0 @@
-.tester{
-    display: flex;
-    justify-content: center;
```

Comment:
Same as this one. If it's blank than it can be deleted. If not at least leave something in here. (even pseudocode so that you know what you INTEND to use